docs/architecture.md

```diff
@@ -1,33 +1,67 @@
-# MiniMint Architecture
+# Minimint Architecture
 
-MiniMint started out as a federated Chaumian e-cash prototype. By now it is a more general framework for federated financial applications. At its core lies the ability to agree on and process transactions. The possible input and output types of these transactions are defined by modules.
+Minimint is a general framework for building federated financial applications.
+The current implementation allows users to make private low-fee payments through a federated blinded mint that issues [e-cash](https://en.wikipedia.org/wiki/Ecash) and processes deposits and withdrawals over Bitcoin or Lightning.
```

Comment:
just a thought: "...allows users for low-fee payments ...".  maybe something like "... allows users to utilize the pegged-in asset (bitcoin), in a cheaper and greatly higher private form (...)" because "low-fee" is so vague, but feel free to ignore or re-phrase. (additional nit "over Bitcoin using On-Chain or Lightning ")

Reply:
I'd like to avoid the peg terminology, rather "backing asset".